### PR TITLE
Added door.model.ts, door-permission.model.ts and door-permissions.co…

### PR DESCRIPTION
**Pull request by TafkaMax**
…mponent.html page where you can see people who have door permissions and what permissions they have.

TODO make the page look nicer
TODO ICT and ICO different color backgrounds

### CODE REVIEW

File: src/environments/environment.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
   production: false,
-  API_URL: 'http://localhost:8080',
+  API_URL: 'https://dev.hub.ituk.ee',
```
Comment (kasparkivistik) [inline src/environments/environment.ts]: soovitan kasutada profiile, siin apiurli ära muutmine võib envi nihu keerata veic

Comment (TafkaMax) [reply src/environments/environment.ts]: tegin uue commiti siia, kuidas seda saab aktiivseks teha?

File: src/app/modules/pages/door-permissions/door-permissions.component.ts [hunk 2]
```diff
@@ -1,19 +1,45 @@
...
+import {NgxSpinnerService} from 'ngx-spinner';
+import {Observable} from 'rxjs';
+import {DoorPermissionModel} from '../../../shared/models/door-permission.model';
+import {DoorHttpService} from '../../../core/http-services/door.http-service';
+import {Router} from '@angular/router';
 
 @Component({
-  selector: 'app-door-permissions',
-  templateUrl: './door-permissions.component.html'
+    selector: 'app-door-permissions',
```
Comment (nils-emil) [inline src/app/modules/pages/door-permissions/door-permissions.component.ts]: This is a page, so selector is not needed. You dont use it inside another component as < app-door-permissions></app-door-permissions> then selector isnt needed.

File: src/app/modules/pages/door-permissions/door-permissions.component.ts [hunk 3]
```diff
@@ -1,19 +1,45 @@
...
-    this.spinner.show();
-    setTimeout(() => {
-      this.spinner.hide();
-    }, 10000);
+    ngOnInit() {
+        this.doorService.getAllUserDoorsServices().subscribe((response) => {
+            this.users = response;
+            this.usersToDisplay = response;
+        }, () => console.log('i dont know how to handle this error :) We will have some monkeys fix this error soon.')),
+            this.spinner.show();
```
Comment (nils-emil) [inline src/app/modules/pages/door-permissions/door-permissions.component.ts]: This spinner is no longer needed imo. the loading spinner is handled globally now. Remove injecting NgxSpinnerService aswell.

File: src/environments/environment.ts (hunk 1 above)
Comment (kasparkivistik) [reply src/environments/environment.ts]: automaatselt läks

**Review by kasparkivistik — APPROVED** [review]

**Comment by kasparkivistik** [comment]
@nils-emil kui ei sobi siis reverti lol

**Pull request merged**